package.json

```diff
@@ -40,9 +40,18 @@
     }
   },
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.2",
     "@babel/runtime": "^7.5.4",
     "@dosomething/forge": "^6.9.1",
     "@dosomething/gateway": "^2.0.1",
+    "@reach/router": "^1.2.1",
```

Comment:
Out of curiosity -- why `@reach/router` over `react-router`?

Reply:
Same question!

Reply:
I'd wanted to play around with it since the API looks a bit cleaner than the one that we're used to, and it was [effectively positioned as a "preview" of upcoming changes to React Router](https://reacttraining.com/blog/reach-react-router-future/).

Buuuuuut after double-checking right now, it seems like the [newer  React Router 5.1 release](https://github.com/ReactTraining/react-router/releases/tag/v5.1.0) with the simplified hooks-based API came out 10 days ago, so I'll just swap to use that!

Reply:
Updated to use `react-router` with the `useParams` hook in 1b2b3eee.